src/main/java/org/asciidoctor/asciidoclet/AsciidoctorConverter.java

```diff
@@ -181,7 +181,21 @@ private String convert(String input, boolean inline) {
         if (input.trim().isEmpty()) {
             return "";
         }
-        options.setDocType(inline ? INLINE_DOCTYPE : "");
+        // Setting doctype to null results in an NPE from asciidoctor.
+        // the default value from the command line is "article".
+        // https://docs.asciidoctor.org/asciidoctor/latest/cli/man1/asciidoctor/#options
+        // In general, in order to respect original doctype, we should do the following.
+        // options.setDocType(inline ?
+        //    INLINE_DOCTYPE :
+        //    options.map().containsKey(Options.DOCTYPE) ?
+        //        Objects.toString(options.map().get(Options.DOCTYPE)) :
```

Comment:
This can be safely cast to String, no need for `Objects.toString`

Reply:
corrected the comment.
done.